app/frontend/src/components/Scroll/_ChevronScroll.scss

```diff
@@ -0,0 +1,62 @@
+@use "../Basics/_colors" as *;
+@use "components/Basics" as *;
+
+.chevron-scroll-outer-container {
+  white-space: nowrap;
+  display: flex;
+  width: 1088px;
+  height: 47px;
+}
+
+.chevron-scroll-child-container {
+  scroll-snap-type: x mandatory;
+  overflow: auto;
+  display: flex;
+  align-items: center;
+  gap: 16px;
+}
+
+.chevron-scroll-child-container::-webkit-scrollbar {
+  display: none;
+}
+
+.chevron-scroll-left-btn {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
```

Comment:
I could be wrong on this as well, but I suspect we already have a white color in our design system. That said, is there a particular reason this needs to be hard-coded in when the browser is white by default?

Reply:
I made the arrows that scroll `button` elements based on research that indicated buttons are preferable to `div` for accessibility. It looks like the default color for buttons is a shade of light gray.

![image](https://github.com/hackforla/CivicTechJobs/assets/73561520/34356378-73b3-4761-8bf2-8275921b8ba1)

Reply:
Ahh, I see! Let's add these and other colors to our basics then